ui/src/app/app.module.ts

```diff
@@ -88,7 +88,12 @@ import {ToolTipComponent} from './views/tooltip/component';
 (<any>window).StackTrace = StackTrace;
 
 function getBasePath() {
-  return localStorage.getItem(overriddenUrlKey) || environment.allOfUsApiUrl;
+  try {
+    return localStorage.getItem(overriddenUrlKey) || environment.allOfUsApiUrl;
+  } catch (err) {
+      console.log('Error accessing local storage: ' + err);
+      return environment.allOfUsApiUrl;
+  }
```

Comment:
I needed to catch this because the service constructor is called before the App component loads.